.travis.yml

```diff
@@ -23,6 +24,7 @@ before_install:
   - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
   - sudo apt-get update -qq
   - sudo apt-get install -y -qq yarn
+  - yarn global add webpack
```

Comment:
Well... I'm agree with this. It's not a big deal and I think it's more safe to run this through yarn.